### DISCUSSION TITLE
Angular 13 with the new @abacritt/angularx-social-login

### DISCUSSION THREAD

**Discussion by ShemiNechmad** [abacritt/angularx-social-login#506] [General]
Dear Heatmanofurioso,
I openend a new project with angular 13 and the new abacritt/angularx-social-login version,
and I'm still getting this error message from google:
"Your client application uses libraries for user authentication or authorization that will soon be deprecated...."

Any thoughts about that?
Thank you.

**Comment by marinescudan** [comment]
@abacritt/angularx-social-login has this provider ```google-login-provider.ts```, which internally loads https://apis.google.com/js/api.js that is being deprecated, read more here:
https://developers.googleblog.com/2022/03/gis-jsweb-authz-migration.html

**Comment by Heatmanofurioso** [comment]
Hi @ShemiNechmad 
There is an open issue for this in https://github.com/abacritt/angularx-social-login/issues/504
And it's being addressed in this PR https://github.com/abacritt/angularx-social-login/pull/507

**Reply by Heatmanofurioso** [reply]
Hey. `@abacritt/angularx-social-login:1.2.1` has been deployed, can you test it with the new version if the lib, please?

**Reply by DavideMoli** [reply]
Hi, @abacritt/angularx-social-login:1.2.1 requires angular 14 as described in [562](https://github.com/abacritt/angularx-social-login/issues/562) . Looks like the latest release supported by angular 13 is 1.1.0

**Comment by MerNat** [comment]
Still getting "Your client application uses libraries for user authentication or authorization that will soon be deprecated...." error from angular 13 with abacritt/angularx-social-login version 1.2.2.

**Comment by heyashwani** [comment]
I am facing same issue in angular 14. i did not find any solution for it on internet,
